internal/service/advancedcluster/model_advanced_cluster.go

```diff
@@ -3,6 +3,7 @@ package advancedcluster
 import (
```

Comment:
can this also happen in cluster resource?

Reply:
I'm not sure, but i saw that priority is an optional attribute there so logic would probably be different